### PR TITLE
[Improve] Harmless changes

### PR DESCRIPTION
**Pull request by dreamerlin**
* Polish docstring (add `None` for argument type and improve indentation)
* Remove redundant `(object)`
* Remove redundant codes
* Update REAMDE.md

### CODE REVIEW

File: configs/recognition/csn/ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb.py [hunk 1]
```diff
@@ -95,7 +95,8 @@
         pipeline=test_pipeline))
 # optimizer
 optimizer = dict(
-    type='SGD', lr=0.0005, momentum=0.9, weight_decay=0.0001)  # 0.0005 for 32g
+    type='SGD', lr=0.0005, momentum=0.9,
+    weight_decay=0.0001)  # this lr is used for 32 gpus
```
Comment (kennymckormick) [inline configs/recognition/csn/ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb.py:99]: We should use the LR for 8 gpus in our config files, even if the model is not trained with 8 gpus.

File: configs/recognition/csn/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb.py [hunk 2]
```diff
@@ -94,7 +94,8 @@
         pipeline=test_pipeline))
 # optimizer
 optimizer = dict(
-    type='SGD', lr=0.0005, momentum=0.9, weight_decay=0.0001)  # 0.0005 for 32g
+    type='SGD', lr=0.0005, momentum=0.9,
+    weight_decay=0.0001)  # this lr is used for 32 gpus
```
Comment (kennymckormick) [inline configs/recognition/csn/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb.py:98]: Same.

File: mmaction/datasets/hvu_dataset.py [hunk 3]
```diff
@@ -56,7 +56,7 @@ class HVUDataset(BaseDataset):
         pipeline (list[dict | callable]): A sequence of data transforms.
         tag_categories (list[str]): List of category names of tags.
         tag_category_nums (list[int]): List of number of tags in each category.
-        filename_tmpl: Template for each filename. `filename_tmpl is None`
+        filename_tmpl (str | None): Template for each filename. If set to None,
```
Comment (kennymckormick) [inline mmaction/datasets/hvu_dataset.py:59]: If set to None, video dataset is used.

File: mmaction/localization/bsn_utils.py [hunk 4]
```diff
@@ -27,13 +27,13 @@ def generate_candidate_proposals(video_list,
...
             model output. Default: '.csv'.
-        result_dict (dict): The dict to save the results. Default: None.
+        result_dict (dict | None): The dict to save the results. Default: None.
 
     Returns:
         dict: A dict contains video_name as keys and proposal list as value.
         If result_dict is not None, save the results to it.
     """
-    if tem_results_ext not in ('.csv'):
+    if tem_results_ext not in ('.csv', ):
```
Comment (kennymckormick) [inline mmaction/localization/bsn_utils.py:36]: if tem_results_ext is not '.csv'

File: mmaction/localization/bsn_utils.py [hunk 5]
```diff
@@ -159,14 +159,14 @@ def generate_bsp_feature(video_list,
...
-        result_dict (dict): The dict to save the results. Default: None.
+        result_dict (dict | None): The dict to save the results. Default: None.
 
     Returns:
         bsp_feature_dict (dict): A dict contains video_name as keys and
         bsp_feature as value. If result_dict is not None, save the
         results to it.
     """
-    if tem_results_ext not in ('.csv') or pgm_proposal_ext not in ('.csv'):
+    if tem_results_ext not in ('.csv', ) or pgm_proposal_ext not in ('.csv', ):
```
Comment (kennymckormick) [inline mmaction/localization/bsn_utils.py:169]: just use `is not`

**Review by kennymckormick — APPROVED** [review]

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
cool

**Pull request merged**